File: java/linear-search/Collection.java
public class Collection {
    private int[] collection; 
    
    public Collection(int[] collection){
        this.collection = collection;
    } 
    
    public int find(int value) {
        for(int interator =0; interator< this.collection.length ; interator++){
            if(this.collection[interator] == value)
                return interator; 
        }
        return -1;
    }
}

File: php/linear-search/collection.php
<?php
class Collection {
    public $array;

    public function __construct ($array){
        $this->array = $array;
    }

    public function find($value){
        $length = count($this->array);
        for ($interator = 0; $interator < $length; $interator++) {
            if($this->array[$interator] == $value){
                return $interator; 
            }
        }
        return -1;
    }
}
?>